akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -100,7 +101,8 @@ object ClusterEvent {
       val seenBy: Set[Address],
       val leader: Option[Address],
       val roleLeaderMap: Map[String, Option[Address]],
-      val unreachableDataCenters: Set[DataCenter])
+      val unreachableDataCenters: Set[DataCenter],
+      @InternalApi private[akka] val memberTombstones: Set[UniqueAddress])
```

Comment:
Perhaps better to make it a private field with a package protected getter...

Reply:
We have to add a binary compatible constructor. Maybe someone is creating their own instances in testing.